### ISSUE TITLE
Global styles: Allow setting of Image styles like border at the Gallery block level

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#48046]
### Description

This issue was found as part of the [FSE Outreach Program's twentieth call for testing](https://make.wordpress.org/test/2023/01/11/fse-program-testing-call-20-find-your-style/#comment-2929):

>It appears that some styles are universal instead of individual categories, which I thought would be the case from how “Style Book” is named. For example, I would think that users would change the borders setting to individual Images only, but the editor applied the same changes to Gallery as well.

While I know the gallery block was refactored to be made up of image blocks to allow for more fine tuned control, I do think it makes sense for these settings to be adjusted either by not allowing certain things in the interface, explaining the connection between the two, or separating out border control options. 

### Step-by-step reproduction instructions

1. Open Appearance > Editor.
2. Click "Edit" to edit the home template. 
3. Open Styles. 
4. Turn on the stylebook by selecting the eye icon.
5. Head to Blocks and notice that if you select image to adjust border that it also adjusts the gallery block below it.

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/26996883/218576471-f599ce44-851f-4cd9-bfff-6e2f69da4526.mov

### Environment info

- WP 6.1.1
- GB 15.1
- TT3

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by annezazu** [comment]
@glendaviesnz since you worked on the gallery block refactor for some thoughts here!

**Comment by glendaviesnz** [comment]
Yes, it would make sense to be able to set the Image block styles at the Gallery level. I have reworded the title slightly, and switched it to enhancement rather than bug - but happy for you to disagree on that and switch it back to bug 😄

**Comment by annezazu** [comment]
Works for me :D
